src/theme.js

```diff
@@ -0,0 +1,20 @@
+const theme = {
+  primary: "#FF4E54",
+  secondary: "#4a4a68",
+  success: "#07CB6E",
+  warning: "#F3B664",
+  cardBorderColor: "#E49393",
+  black: "#060047",
+  cardTextColor: "#1640D6",
```

Comment:
color name should not give like as per component it should be as per color name like dark-gray, yellowGreen, lightBlack @Arbina001

Reply:
ok sir I have change it in main file